tailwind.config.js

```diff
@@ -45,10 +45,13 @@ module.exports = {
         xs: '9px',
         sm: '10px',
         md: '18px',
+        'md-40': '40px',
         xl: '200px',
         'xl1': '240px',
```

Comment:
You really don't need it. It's bad that your size will be a little different, better to organize it universally...

h-xs = h-2 (8px)
h-xl ~= h-48 (192px)
h-xl1 ~= h-64 (256px)
...